js/options.js

```diff
@@ -58,4 +58,31 @@
             });
         });
     });
+
+    // Apply i18n
+    $(document).ready(function(){
+        // Basic Substitution
```

Comment:
Rather than jquery all the things, we should probably convert most of the html markup for this page to a Mustache template that the InstallView can render with i18n'd content.